modules/RTC/ortc/RTCPeerConnection.js

```diff
@@ -672,18 +1052,837 @@ export default class ortcRTCPeerConnection extends yaeti.EventTarget {
+     */
+    _sendMedia() {
+        logger.debug('_sendMedia()');
+
+        for (const info of this._localTrackInfos.values()) {
+            // Ignore if already sending.
+            if (info.sending) {
+                continue; // eslint-disable-line no-continue
+            }
+
+            // Update sending field.
+            info.sending = true;
+
+            const rtpSender = info.rtpSender;
+            const ssrc = info.ssrc;
+            const rtxSsrc = info.rtxSsrc;
+            const track = rtpSender.track;
+            const kind = track.kind;
+
+            const parameters = this._getParametersForRtpSender(kind, {
+                ssrc,
+                rtxSsrc
+            });
+
+            logger.debug(
+                'calling rtpSender.send(), parameters:', parameters);
+
+            // Start rsending media.
+            try {
+                rtpSender.send(parameters);
+            } catch (error) {
+                logger.error(`rtpSender.send() failed:${error}`);
+            }
+        }
+    }
+
+    /**
+     * Start receiving remote media.
+     */
+    _receiveMedia() {
+        logger.debug('_receiveMedia()');
+
+        const currentRemoteSsrcs = new Set(this._remoteTrackInfos.keys());
+        const newRemoteTrackInfos
+            = sdpUtils.extractTrackInfos(this._remoteDescription.sdpObject);
+
+        // Map of new remote MediaStream indexed by MediaStream.jitsiRemoteId.
+        const addedRemoteStreams = new Map();
+
+        // Map of remote MediaStream indexed by added MediaStreamTrack.
+        // NOTE: Just filled for already existing streams.
+        const addedRemoteTracks = new Map();
+
+        // Map of remote MediaStream indexed by removed MediaStreamTrack.
+        const removedRemoteTracks = new Map();
+
+        logger.debug(
+            '_receiveMedia() remote track infos:', newRemoteTrackInfos);
+
+        // Check new tracks.
+        for (const pair of newRemoteTrackInfos) {
+            const ssrc = pair[0];
+            const info = pair[1];
+
+            // If already handled, ignore it.
+            if (currentRemoteSsrcs.has(ssrc)) {
+                continue; // eslint-disable-line no-continue
+            }
+
+            logger.debug(`_receiveMedia() new remote track, ssrc:${ssrc}`);
+
+            // Otherwise append to the map.
+            this._remoteTrackInfos.set(ssrc, info);
+
+            const kind = info.kind;
+            const rtxSsrc = info.rtxSsrc;
+            const streamRemoteId = info.streamId;
+            const trackRemoteId = info.trackId;
+            const cname = info.cname;
+            const isNewStream = !this._remoteStreams.has(streamRemoteId);
+            let stream;
+
+            if (isNewStream) {
+                logger.debug(
+                    `_receiveMedia() new remote stream, id:${streamRemoteId}`);
+
+                // Create a new MediaStream.
+                stream = new MediaStream();
```

Comment:
Actually linters did not complain. I expect `MediaStream` is already well known stuff.